### PR TITLE
Upgrade to php7.3

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
- *PHP version not set*
- *#83 Allow to choose language at installation*

## Solution
- *Upgrade to PHP7.3*
- *Add language to choose at installation #83*
- *Set Elastic theme as default"*
- *Disable GoogleSpell as default*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** : 
- [ ] **Approval (LGTM)** : 
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/roundcube_ynh%20PR-NUM-/)  
*Please replace '-NUM-' in this link by the PR number.*  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/roundcube_ynh%20PR96%20(ericgaspar)/)

### CODE REVIEW

**Review by yalh76 — APPROVED** [review]
kick review ^^

File: scripts/upgrade [hunk 1]
```diff
@@ -158,17 +153,18 @@ then
     ynh_replace_string --match_string="__DBUSER__" --replace_string=$db_name --target_file="$rc_conf"
     db_pwd=$(ynh_app_setting_get --app=$app --key=mysqlpwd)
     ynh_replace_string --match_string="__DBPASS__" --replace_string="$db_pwd" --target_file="$rc_conf"
-    ynh_replace_string --match_string="__DBNAME__" --replace_string=$db_name --target_file="$rc_conf"
+    ynh_replace_string --match_string="__DBNAME__" --replace_string="$db_name" --target_file="$rc_conf"
+    ynh_replace_string --match_string="__LANGUAGE__" --replace_string="$language" --target_file="$rc_conf"
```
Comment (yalh76) [inline scripts/upgrade:157]: You need to manage upgrade from previous version when `ynh_app_setting_get --app=$app --key=language` don't already exist

**Review by yalh76 — CHANGES_REQUESTED** [review]

File: scripts/_common.sh [hunk 2]
```diff
@@ -4,8 +4,10 @@
 # COMMON VARIABLES
 #=================================================
 
+YNH_PHP_VERSION="7.3"
+
 # Package dependencies
-pkg_dependencies="php-cli php-common php-intl php-json php-pear php-auth-sasl php-mail-mime php-patchwork-utf8 php-net-smtp php-net-socket php-zip php-gd php-mbstring php-curl"
+extra_php_dependencies="php-pear php${YNH_PHP_VERSION}-ldap php${YNH_PHP_VERSION}-mysql php${YNH_PHP_VERSION}-cli php${YNH_PHP_VERSION}-intl php${YNH_PHP_VERSION}-json php${YNH_PHP_VERSION}-zip php${YNH_PHP_VERSION}-gd php${YNH_PHP_VERSION}-mbstring php${YNH_PHP_VERSION}-dom php${YNH_PHP_VERSION}-curl"
```
[the remark this answers is not in the document]
Comment (ericgaspar) [reply scripts/_common.sh:10]: Does it work? I can't see a version 7.3 for pear in debian repo https://packages.debian.org/fr/buster/php-pear.
I will try

File: check_process [hunk 3]
```diff
@@ -12,7 +12,6 @@
 		setup_private=0
 		setup_public=0
 		upgrade=1
-		upgrade=1 from_commit=8d0acb6953a4c4c2206985c7d9881c5b7b637ebd
```
[the remark this answers is not in the document]
Comment (ericgaspar) [reply check_process:15]: I guess with Composer upgrading to last version, all version prior to this one will fail

File: scripts/_common.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (yalh76) [reply scripts/_common.sh:10]: I don't know... I only assume...

File: check_process (hunk 3 above)
[the remark this answers is not in the document]
Comment (yalh76) [reply check_process:15]: Very good point... It was out of my head ...

File: scripts/upgrade (hunk 1 above)
Comment (ericgaspar) [reply scripts/upgrade:157]: @yalh76 Any hits on best way to do that? 🤔

Comment (yalh76) [reply scripts/upgrade:157]: you can do like it's made for `$db_name`
https://github.com/YunoHost-Apps/roundcube_ynh/blob/f6183f057154ce38c5004f7056a7189536d71a04/scripts/upgrade#L37-L41

**Review by yalh76 — APPROVED** [review]

**Pull request merged**